assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -294,9 +311,49 @@ const SearchFunnelWidget = ( {
 	}
 
 	if ( isSearchConsoleGatheringData ) {
+		const halfCellProps = {
+			smSize: 4,
+			mdSize: 4,
+			lgSize: 6,
+		};
+
 		return (
 			<Widget Header={ Header } Footer={ WidgetFooter }>
-				<WidgetReportZero moduleSlug="search-console" />
+				{ isAnalyticsConnected && isAnalyticsActive && (
+					<WidgetReportZero moduleSlug="search-console" />
+				) }
+
+				{ ( ! isAnalyticsConnected || ! isAnalyticsActive ) && (
+					<Grid>
+						<Row>
+							<Cell { ...halfCellProps }>
+								<ReportZero moduleSlug="search-console" />
+							</Cell>
+
+							{ ! isNavigatingToReauthURL && (
+								<Cell { ...halfCellProps }>
+									{ ! isAnalyticsActive && (
+										<ActivateModuleCTA moduleSlug="analytics" />
+									) }
+
+									{ isAnalyticsActive &&
+										! isAnalyticsConnected && (
+											<CompleteModuleActivationCTA moduleSlug="analytics" />
+										) }
```

Comment:
Maybe this could be simplified using something like `whenActive( { moduleName: 'analytics', FallbackComponent, IncompleteComponent } )( Null )` ?